pkg/kubelet/util/store/store.go

```diff
@@ -0,0 +1,53 @@
+/*
+Copyright 2017 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package store
+
+import (
+	"fmt"
+	"regexp"
+)
+
+const keyMaxLength = 250
+
+var keyRegex = regexp.MustCompile("^[a-zA-Z0-9]+$")
```

Comment:
Maybe add some GoDoc on the rules for key names

Reply:
Add a comment describing the key regex. Also changed it to be consistent with the qualified name in kubernetes api.